src/resources/Resource.js

```diff
@@ -58,7 +60,16 @@ export default class Resource {
   }
 
   async parseResponse (response) {
-    const data = await response.json()
+    let text = await response.text()
```

Comment:
Can you comment here justifying the existence of this block? Surely there's a better way than overridding [`Body.json()`](https://developer.mozilla.org/en-US/docs/Web/API/Body/json)